packaging/dbscripts/upgrade/pre_upgrade/0000_config.sql

```diff
@@ -1311,7 +1311,7 @@ select fn_db_update_config_value('ServerCPUList',
 select fn_db_update_config_value('AgentAppName','ovirt-guest-agent-common,ovirt-guest-agent,qemu-guest-agent','general');
 
 select fn_db_update_config_value('CertExpirationAlertPeriodInDays', '30', 'general');
-select fn_db_update_config_value('CertExpirationWarnPeriodInDays', '120', 'general');
+select fn_db_update_config_value('CertExpirationWarnPeriodInDays', '365', 'general');
```

Comment:
wouldn't it make sense to add all those things to engine-config as well?

Reply:
it would be best, yes please

Reply:
+1